frontend/dist/assets/branding.sample/welcome-simple-text.html

```diff
@@ -5,7 +5,7 @@ <h2 style="margin-bottom: 1em; color: #555">Collaborative information management
   management that has been created over 20 years of experience in the service of collective intelligence.
 </p>
 <p>
-  Tracim is available in: Deutsch, English, Español, Français, Português and العربية
+  Tracim is available in: Deutsch, English, Español, Français, Norsk, Português and العربية
```

Comment:
I don't know how this render in a screen, however, maybe it will be more readable in a list?